src/manifest.json

```diff
@@ -2,6 +2,7 @@
   "name": "Superhero",
   "description": "Superhero Wallet",
   "manifest_version": 2,
+  "content_security_policy": "default-src 'self' 'unsafe-inline'; script-src 'self' 'sha256-BCCVwoZgDUk/cw+Ln2/t3z9bm8qy+sICaZP5BWQIyUg='; connect-src *; font-src * data:; img-src * data:; style-src-elem *",
```

Comment:
```
Refused to execute inline script because it violates the following 
Content Security Policy directive: "script-src 'self' 'sha256-BCCVwoZgDUk/cw+Ln2/t3z9bm8qy+sICaZP5BWQIyUg='". Either the 'unsafe-inline' keyword, a hash ('sha256-LDP+OjgDpfArTDlfhpqHPU5mcov+qbvbnr657nRMBZY='), or a nonce ('nonce-...') is required to enable inline execution
Uncaught EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following 
Content Security Policy directive: "script-src 'self' 'sha256-BCCVwoZgDUk/cw+Ln2/t3z9bm8qy+sICaZP5BWQIyUg='".
```

Reply:
Are you executing `npm run build:extension` or `npm run watch`? When building for development webpack is using `eval()` which is not allowed
`npm run build:extension` should work